quirks/password-rules.json

```diff
@@ -167,6 +167,9 @@
     "bitly.com": {
         "password-rules": "minlength: 6; required: lower; required: upper; required: digit; required: [`!@#$%^&*()+~{}'\";:<>?]];"
     },
+    "bkvenergy.com": {
+        "password-rules": "minlength: 8; maxlength: 12; required: upper; required: lower; required: digit; required: [~!@#$%^&*()_=+,<.>- ];"
```

Comment:
Thank you for this contribution! It looks good! One thing needs to be changed, tho. Character classes have two tricky rules:

1. `-` needs to be the first character in a class if it’s included.
2. `]` needs to be the last character in a class if it’s included. (Doesn’t affect this pull request, but I figured I’d let you know for fun!)